test/NoRecommendationProvided/cases/Under25NilmHpvNegative.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Under 25 NILM HPV Negative
+
+data:
+- 
+  resourceType: Patient
+  name: Joanne Smith
+  gender: female
+  birthDate: 1998-05-01
+  extension:
+  -
+    url: http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex
+    valueCode: F
+
+- 
+  resourceType: DiagnosticReport
+  code: LOINC#10524-7 Microscopic observation [Identifier] in Cervix by Cyto stain
+  status: amended
+  conclusionCode:
+  - SNOMEDCT#373887005 Negative for intraepithelial lesion or malignancy (finding)
+  effectiveDateTime: 2021-01-02
+- 
+  resourceType: DiagnosticReport
+  code: LOINC#21440-3 Human papilloma virus 16+18+31+33+35+45+51+52+56 DNA [Presence] in Cervix by Probe
+  status: final
+  conclusionCode:
+  - SNOMEDCT#787724008 Human papillomavirus deoxyribonucleic acid test negative (finding)
+  effectiveDateTime: 2021-01-02
+
+# This is meant to fail CDS recommendation
```

Comment:
In the cqlt.yaml file, this test case only points to the ManagementLibrary. However if you ran this against the screening library, I believe that there would be a recommendation from the average risk screening library for this case, since the patient would have had no abnormal results to bring them over to management. Therefore I propose we may want to delete this test case, since this does not actually reflect what we would expect the overall CDS to return, since they _would_ be getting a recommendation from the AverageRiskScreening libary. I think this would be the same for Under25NilmHpvUnkown case, but it would be worth checking by running them against the screening library.

Reply:
I wanted a test which would NOT return the NILM/HPV recommendation, demonstrating the effect of a negative HPV. It could be moved as you suggest. Same for below.

Reply:
I see what you mean. On second thought I think having this test alongside the Under25HpvPos makes sense to show what the difference between the two would be. I am fine with keeping it.